packages/documentation/src/content/docs/concepts/accounting/accounts-and-transfers.md

```diff
@@ -751,12 +764,12 @@ Sender and receiver do not have wallet addresses at the same Rafiki instance.
 | -------------- | -------------- |
 | Peer Liquidity | Wallet Address |
 
-- Example: A wallet address receives 2 USD from an outgoing payment at a peer's Rafiki instance.
+- Example: A wallet address receives `2 USD` from an outgoing payment at a peer's Rafiki instance.
```

Comment:
I don't think "from an outgoing payment" is correct here, since SPSP is more like direct/raw ILP access instead